libs/common/src/index.ts

```diff
@@ -4,3 +4,4 @@ export * from './helpers';
 export * from './sql-db-entities';
 export * from './test-helpers';
 export * from './types';
+export * from './dtfs-central-api-response-types';
```

Comment:
Is the idea that these will also be used in the other services that call `dtfs-central-api` endpoints?

Reply:
Yes, lots of endpoints are called by tfm api and portal api so this made sense a place to put the source of truth for response types.